src/System.Drawing.Common/src/System/Drawing/Font.Serializable.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Drawing
+{
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.Runtime.Serialization;
+    using System.Security.Permissions;
```

Comment:
Same general comments as in the first file.  I won't comment on these elsewhere.